packages/twenty-front/src/modules/ui/input/components/internal/date/components/DateTimeInput.tsx

```diff
@@ -37,20 +41,27 @@ export const DateTimeInput = ({
   onChange,
   isDateTimeInput,
 }: DateTimeInputProps) => {
+  const parsingFormat = isDateTimeInput ? 'MM/dd/yyyy HH:mm' : 'MM/dd/yyyy';
+
   const [hasError, setHasError] = useState(false);
 
-  const parseDateToString = (date: any) => {
-    const dateParsed = DateTime.fromJSDate(date);
+  const parseDateToString = useCallback(
+    (date: any) => {
+      const dateParsed = DateTime.fromJSDate(date);
 
-    const formattedDate = dateParsed.toFormat('MM/dd/yyyy HH:mm');
+      const formattedDate = dateParsed.toFormat(parsingFormat);
 
-    return formattedDate;
-  };
+      return formattedDate;
+    },
+    [parsingFormat],
+  );
 
   const parseStringToDate = (str: string) => {
     setHasError(false);
 
-    const parsedDate = DateTime.fromFormat(str, 'MM/dd/yyyy HH:mm');
+    const parsedDate = DateTime.fromFormat(str, parsingFormat, {
+      zone: isDateTimeInput ? undefined : 'utc',
```

Comment:
Question: undefined here means it will use the implied timezone from the string? (which should contain the timezone now), right?

Reply:
I'll rewrite it so it's more clear.